ale/drivers/base.py

```diff
@@ -103,15 +409,28 @@ def line_scan_rate(self):
     @property
     def number_of_ephemerides(self):
         #TODO: Not make this hardcoded
-        return 909
+        return 909 #len(self._sensor_position)
```

Comment:
Is there a reason why this is returning the hardcoded value with the real value commented out?

Reply:
Right now the way things are set up, returning the real value ends with a stack overflow. self._sensor_positions - calls -> self.number_of_ephemerides - calls -> self._sensor_position and so on. Didn't seem to be an immediately obvious solution so I left it in.

Reply:
What about the constructor taking this as an argument? We can/should write the logic to compute a reasonable number of ephemerides if a user specified number is not provided.

Reply:
definitely better than hardcoded.

Reply:
The number of ephemerides is going to be based on the length and jitter of the line scan image. A reasonabl-ish default would be 1 per line. That is probably overkill, though, and we can get away with 1 per 5-10 lines.

Reply:
I figure either some function of the line number or (assuming Spice just has a table of values which it interpolates on) the number of positions/quats in the spice table.